akka-stream-tests/src/test/scala/akka/stream/io/TlsSpec.scala

```diff
@@ -102,469 +102,486 @@ class TlsSpec extends StreamSpec(TlsSpec.configOverrides) with WithLogCapturing
   import system.dispatcher
 
   "SslTls" must {
+    "work for TLSv1.2" must { workFor("TLSv1.2", TLS12Ciphers) }
 
-    val sslContext = initSslContext()
+    if (JavaVersion.majorVersion >= 11)
+      "work for TLSv1.3" must { workFor("TLSv1.3", TLS13Ciphers) }
 
-    val debug = Flow[SslTlsInbound].map { x =>
-      x match {
-        case SessionTruncated   => system.log.debug(s" ----------- truncated ")
-        case SessionBytes(_, b) => system.log.debug(s" ----------- (${b.size}) ${b.take(32).utf8String}")
+    def workFor(protocol: String, ciphers: Set[String]): Unit = {
+      val sslContext = initSslContext(protocol)
```

Comment:
You can hide whitespace changes for this file when reviewing, mostly only indentation changed.